lib/debugger/debugger.dart

```diff
@@ -1232,11 +1232,16 @@ class BreakOnExceptionControl extends CoreElement {
     _allElement = all.element;
 
     add([
-      span(text: 'Break on: '),
       new CoreElement('label')
-        ..add(<CoreElement>[unhandled, span(text: ' Unhandled exceptions')]),
+        ..add(<CoreElement>[
```

Comment:
It could be nice to add icons for these in the future so we can just show an icon when the view width is small.

Reply:
Icons can be difficult to learn; perhaps - when we look to better support narrow layouts - we should investigate converting to and from an overflow style menu?

Reply:
agree this text ends up being very wide...